cpp/modules/deck.gl/core/src/shaderlib/project/viewport-uniforms.cpp

```diff
@@ -89,6 +89,8 @@ auto getOffsetOrigin(const std::shared_ptr<Viewport>& viewport, COORDINATE_SYSTE
       offsetMode = false;
   }
 
+  // TODO(ilija@unfolded.ai): Aren't designated initializers part of C++20?
+  // https://en.cppreference.com/w/cpp/language/aggregate_initialization
```

Comment:
Even though cppreference is my goto source for C++ standard questions, this article doesn't really make it clear in my opinion.

I do find a lot of articles like: https://mariusbancila.ro/blog/2020/02/27/c20-designated-initializers/

If this is already supported by both clang and gcc, let's use it, but we should probably call out in our C++ coding standard that we use C++17 with selected C++20 features.